astropy/utils/tests/test_introspection.py

```diff
@@ -58,12 +58,6 @@ def test_find_mod_objs():
 
     # this import  is after the above call intentionally to make sure
     # find_mod_objs properly imports astropy on its own
-    import astropy
-
-    # just check for astropy.test ... other things might be added, so we
-    # shouldn't check that it's the only thing
-    assert "test" in lnms
-    assert astropy.test in objs
```

Comment:
this was testing the output of `find_mod_objs` so should probably be replaced by another astropy subpackage?